### ISSUE TITLE
[Feature]: Add mulitple item to the cart

### ISSUE THREAD

**Issue by Mmasson-01** [vuestorefront/shopify#70]
### How the project can be improved?

It is only possible to add one item at a time utilizing the **Composables** from the Vuestorefront Shopify integrations.

Reference : https://docs.vuestorefront.io/shopify/guide/use-cart.html#api

This bring an incoveniance the moment we want to add multiple item before doing anything else.

It would be interesting if it was possible to add another composable, such as **addItems** taking an array of Product instead of a single Product.

### What are the acceptance criteria?

- [ ] Takes an array of object in parameters
- [ ] Is asynchronous
- [ ] Is possible to wait for every item to be added to the cart before continuing the execution
- [ ] Push errors, just like the **addItem** method
- [ ] Push results, just like the **addItem** method

### Additional information

Here an exemple of workaround we currently use in our project : 

```js
  const addItems = (items: any[]): Promise<AddItemResolveType> => {
    loading.value = true;
    return new Promise((resolve) => {
      items.reduce((p, product, index) => {
        return p.then(async () => {
          await addItem({ product, quantity: 1 });
          if (error.value.addItem) {
            errors.value.push(product);
          } else {
            result.value.push(product);
          }
          if (hasLoopOverAllItems(items, index)) {
            loading.value = false;
            resolve({ result: result.value, errors: errors.value });
          }
        });
      }, Promise.resolve());
    });
  };
```

An exemple of use case would be : 

- Opening a side panel as a Buy modal
- Checking multiple item to add to the cart
- Clicking on a "add to cart" button
- Waiting for items to be added
- Closing the Buy Modal to bring up the Cart when addItems is complete

Opening a feature suggestion is a first for me. My apologies if any information is missing.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aureate-labs-team** [comment]
We're working on it

**Comment by jeffpdotone** [comment]
Closes by #151
